Source/WebCore/platform/network/cocoa/NetworkStorageSessionCocoa.mm

```diff
@@ -429,21 +429,21 @@ static void deleteAllHTTPCookies(CFHTTPCookieStorageRef cookieStorage)
     if (!cookie)
         return nil;
 
+    NSMutableDictionary* properties = [[cookie properties] mutableCopy];
+    [properties setValue:@"True" forKey:@"SetInJavaScript"];
+    NSHTTPCookie* cookieSetInJavaScript = [NSHTTPCookie cookieWithProperties:properties];
```

Comment:
CFNetwork is going to expose setInJavascript as an SPI on NSHTTPCookie so we don't need to create the cookie twice here.